connector/connectorRunner.go

```diff
@@ -114,6 +117,7 @@ func (cr *connectorRunner) Run() error {
 		gogoProtoMarshaller,
 		blocksPool,
 		outportBlockConverter,
+		firstCommitableBlocks,
```

Comment:
you are giving here the same map as for publisher
The map is not changed in either publisher or data processor right?

Reply:
yes, the map is not being changed inside

Reply:
updated to use deep copy here also